Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -174,27 +176,41 @@ public static AuthenticationBuilder AddCustomAuthentication(this IServiceCollect
         /// </summary>
         /// <param name="serviceCollection">The service collection.</param>
         /// <param name="pluginAssemblies">An IEnumerable containing all plugin assemblies with API controllers.</param>
-        /// <param name="knownProxies">A list of all known proxies to trust for X-Forwarded-For.</param>
+        /// <param name="config">The <see cref="NetworkConfiguration"/>.</param>
         /// <returns>The MVC builder.</returns>
-        public static IMvcBuilder AddJellyfinApi(this IServiceCollection serviceCollection, IEnumerable<Assembly> pluginAssemblies, IReadOnlyList<string> knownProxies)
+        public static IMvcBuilder AddJellyfinApi(this IServiceCollection serviceCollection, IEnumerable<Assembly> pluginAssemblies, NetworkConfiguration config)
         {
             IMvcBuilder mvcBuilder = serviceCollection
                 .AddCors()
                 .AddTransient<ICorsPolicyProvider, CorsPolicyProvider>()
                 .Configure<ForwardedHeadersOptions>(options =>
                 {
+                    options.ForwardLimit = 3;
```

Comment:
This should be configurable

Reply:
It should just be set to null if known proxies are configured

Reply:
Really good explanation here: 
https://blog.novanet.no/parameter-count-mismatch-between-x-forwarded-for-and-x-forwarded-proto/

Have set to null. Can't think of one myself, but could there be any instance whereby the server would need to obtain one of the proxy addresses instead of the clients ?